shared/kubernetes/kubernetes.go

```diff
@@ -87,3 +88,26 @@ func guessIngress() string {
 
 	return ingress
 }
+
+// RestartService restarts the pod.
+func RestartService(ServerFilter string) error {
+	if err := StopService(ServerFilter); err != nil {
+		return fmt.Errorf("cannot stop %s: %s", ServerFilter, err)
+	}
+	return StartService(ServerFilter)
+}
+
+// StartService starts the pod.
+func StartService(ServerFilter string) error {
```

Comment:
The word `Service` doesn't fit the kubernetes world and is even misleading.

```suggestion
// Start starts the pod.
func Start(ServerFilter string) error {
```